tools/ci_build/github/linux/run_dockerbuild.sh

```diff
@@ -83,10 +83,16 @@ elif [ $BUILD_DEVICE = "gpu" ]; then
         $GET_DOCKER_IMAGE_CMD --repository "onnxruntime-$IMAGE" \
             --docker-build-args="--build-arg BASEIMAGE=nvcr.io/nvidia/cuda:11.1.1-cudnn8-devel-${BUILD_OS} --build-arg BUILD_USER=onnxruntimedev --build-arg BUILD_UID=$(id -u) --build-arg PYTHON_VERSION=${PYTHON_VER} --build-arg INSTALL_DEPS_EXTRA_ARGS=\"${INSTALL_DEPS_EXTRA_ARGS}\" --build-arg USE_CONDA=${USE_CONDA} --network=host" \
             --dockerfile Dockerfile.ubuntu_gpu_training --context .
-elif [ $BUILD_DEVICE = "tensorrt" ]; then
-        # TensorRT container release 20.12
-        IMAGE="$BUILD_OS-cuda11.1-cudnn8.0-tensorrt7.2"
-        DOCKER_FILE=Dockerfile.ubuntu_tensorrt
+elif [[ $BUILD_DEVICE = "tensorrt"* ]]; then
```

Comment:
We will need to stop relying on TensorRT ubuntu images in the near future.
it doesn't allow us to control the cuda versions etc.
The problem will come when we combine CUDA and TensorRT in the same package.  CUDA uses CentOS builds.

Reply:
So it seems that we need to create and maintain our own TensorRT ubuntu image in order to control TRT/CUDA versions?
(we can reference from https://github.com/NVIDIA/TensorRT/blob/master/docker/ubuntu-18.04.Dockerfile)

For CUDA uses CentOS builds, you mean CUDA related libraries in the docker image nvcr.io/nvidia/tensorrt:20.07.1-py3 were built on CentOS?

Reply:
building our own ubuntu image would work too.
but it might make more sense to switch to CentOS because that's what our CUDA pipelines use.
see https://github.com/microsoft/onnxruntime/blob/master/tools/ci_build/github/azure-pipelines/linux-gpu-ci-pipeline.yml#L16
BASEIMAGE uses nvcr.io/nvidia/cuda:11.1.1-cudnn8-devel-centos7
we can install TensorRT dependencies like this:
https://github.com/microsoft/onnxruntime/blob/master/tools/ci_build/github/linux/docker/Dockerfile.manylinux2014_cuda11_1_tensorrt7_2#L181

Reply:
Changed to use CentOS image and installed TensorRT 7.1 dependencies.

This modification has been tested.
https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=424158&view=results

Reply:
> Changed to use CentOS image and installed TensorRT 7.1 dependencies.
> 
> This modification has been tested.
> https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=424158&view=results

That's great. I see there are still comments referring to TensorRT container 20.07 and 20.12  (which are Ubuntu)
We'll need to remove those comments to avoid confusion.

Reply:
I'm assuming you are referring to line88 and line92 of run_dockerbuild.sh.
It's previous way of building TRT EP docker image by calling run_dockerbuild.sh and run_build.sh with Dockerfile.ubuntu_tensorrt7_1.

Reply:
I saw you added the new way of using manylinux docker image, as shown in linux-gpu-tensorrt-7-1-ci-pipeline.yml and Dockerfile.manylinux2014_cuda11_tensorrt7_1.

Reply:
ok yes, let's keep old way. thanks!